src/hooks/useActiveDescendantListFocus/useActiveDescendantListFocusHooks.ts

```diff
@@ -100,14 +109,23 @@ export function useSupportPressItemKeyboardNavigation({
   isItemSelectable,
   listenerOptions = undefined,
   isIgnoreSpaceAsItemSelection = false
+}: {
+  visualFocusItemIndex: number;
+  focusedElementRef: MutableRefObject<HTMLElement>;
+  itemsCount: number;
+  setVisualFocusItemIndex: (index: number) => unknown;
```

Comment:
same comment about `void` vs `unknown` return type
```suggestion
  setVisualFocusItemIndex: (index: number) => void;
```

Reply:
@laviomri same answer :)